app/views/results/common/_peer_reviews.html.erb

```diff
@@ -12,15 +12,16 @@
       <% prs.each_with_index do |pr, i| %>
         <% if @current_role.student? %>
           <li> <%= link_to("#{PeerReview.model_name.human} #{i + 1}",
-                        view_marks_assignment_result_path(assignment.id, pr.result_id)) %> </li>
+                        view_marks_course_result_path(assignment.id, pr.result_id)) %> </li>
         <% else %>
             <% pr_result = pr.result %>
             <% group_name = pr_result.submission.grouping.group.group_name %>
             <li> <%= link_to(group_name,
-                             view_marks_assignment_submission_result_path(assignment_id: assignment.id,
-                                                                          submission_id: submission.id,
-                                                                          id: pr.result_id,
-                                                                          reviewer_grouping_id: reviewer.id)) %> </li>
+                             view_marks_course_assignment_submission_result_path(course_id: @current_course.id,
```

Comment:
This route helper method is not right. Result member routes are only nested under course